usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -0,0 +1,406 @@
+        max_id = self.spark.sql("SELECT MAX(id) AS max_id FROM int.award_id_lookup").collect()[0]["max_id"]
+        with connection.cursor() as cursor:
+            cursor.execute(f"SELECT setval('award_id_seq', {max_id})")
+
+    def delete_records_sql(self):
+        if self.etl_level == "transaction_id_lookup":
+            id_col = "id"
+            subquery = """
+                SELECT id AS id_to_remove
+                FROM int.transaction_id_lookup AS tidlu LEFT JOIN raw.detached_award_procurement AS dap ON (
+                    tidlu.detached_award_procurement_id = dap.detached_award_procurement_id
+                )
+                WHERE tidlu.detached_award_procurement_id IS NOT NULL AND dap.detached_award_procurement_id IS NULL
+                UNION ALL
+                SELECT id AS id_to_remove
+                FROM int.transaction_id_lookup AS tidlu LEFT JOIN raw.published_fabs AS pfabs ON (
+                    tidlu.published_fabs_id = pfabs.published_fabs_id
+                )
+                WHERE tidlu.published_fabs_id IS NOT NULL AND pfabs.published_fabs_id IS NULL
+            """
+        elif self.etl_level == "award_id_lookup":
+            id_col = "transaction_unique_id"
+            subquery = """
+                SELECT aidlu.transaction_unique_id AS id_to_remove
+                FROM int.award_id_lookup AS aidlu LEFT JOIN raw.detached_award_procurement AS dap ON (
+                    aidlu.detached_award_procurement_id = dap.detached_award_procurement_id
+                )
+                WHERE aidlu.detached_award_procurement_id IS NOT NULL AND dap.detached_award_procurement_id IS NULL
+                UNION ALL
+                SELECT aidlu.transaction_unique_id AS id_to_remove
+                FROM int.award_id_lookup AS aidlu LEFT JOIN raw.published_fabs AS pfabs ON (
+                    aidlu.published_fabs_id = pfabs.published_fabs_id
+                )
+                WHERE aidlu.published_fabs_id IS NOT NULL AND pfabs.published_fabs_id IS NULL
+            """
+
+        sql = f"""
+            MERGE INTO int.{self.etl_level}
+            USING (
+                {subquery}
+            ) AS deleted_records
+            ON {self.etl_level}.{id_col} = deleted_records.id_to_remove
+            WHEN MATCHED
+            THEN DELETE
+        """
+        return sql
+
+    def initial_run(self):
+        """
+        TODO:
+            - External Data values need to be added for all etl_levels (including transaction_ids)
+            - Initial value of the sequence should be set
+        """
+        #### transaction_id_lookup
+
+        # Capture start time of the transaction_id_lookup creation to update the "last_load_date" after completion
+        transaction_id_lookup_start_time = datetime.now(timezone.utc)
+
+        spark_s3_bucket = CONFIG.SPARK_S3_BUCKET
+        delta_lake_s3_bucket = CONFIG.DELTA_LAKE_S3_PATH
+        destination_database = "int"
```

Comment:
Can you move the common config (`spark_s3_bucket`, `delta_lake_s3_bucket`, and `destination_database`) above the `transaction_id_lookup` section?